packages/libs/react-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  containerClass,
+  disabledClass,
+  outlinedClass,
+  textAreaClass,
+  textAreaContainerClass,
+} from './TextArea.css';
+
+import type { Sprinkles } from '@theme/sprinkles.css';
+import { sprinkles } from '@theme/sprinkles.css';
+import classNames from 'classnames';
+import type { FC, TextareaHTMLAttributes } from 'react';
+import React, { forwardRef, useState } from 'react';
+
+export interface ITextareaProps
+  extends Omit<
+      TextareaHTMLAttributes<HTMLTextAreaElement>,
+      'as' | 'disabled' | 'children' | 'className' | 'id'
+    >,
+    Partial<Pick<Sprinkles, 'fontFamily'>> {
+  disabled?: boolean;
+  ref?: React.ForwardedRef<HTMLTextAreaElement>;
+  id: string;
+  outlined?: boolean;
+}
+
+export const Textarea: FC<ITextareaProps> = forwardRef<
+  HTMLTextAreaElement,
+  ITextareaProps
+>(function TextArea({ outlined, disabled = false, fontFamily, ...rest }, ref) {
+  const [value, setValue] = useState<string>('');
+
+  const handleValueChange = ({
+    target,
+  }: React.ChangeEvent<HTMLTextAreaElement>): void => {
+    setValue(target.value);
+  };
```

Comment:
This should be coming from the props, not internal state, otherwise users of the TextArea component will not able to control the value or in this case, get the updated value